### PR TITLE
New Extension: Shell Buddy

### PR DESCRIPTION
**Pull request by JohnBra**
## Description

An extension to convert natural language prompts to runnable shell commands. You can either pass the prompt directly as an argument, or convert prompts within the extension command.

Every converted command will be appended to the command history and can be copied to the clipboard, or executed in the terminal.

You will need a license key with credits to convert prompts, since the extension is using an AI API.

## Screencast

https://user-images.githubusercontent.com/36050523/194008741-803d767e-a825-427d-9669-78b5263454ba.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/shell-buddy/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Convert Prompt",
+      "subtitle": "GPT-3",
```
Comment (pernielsentikaer) [inline extensions/shell-buddy/package.json:17]: GPT-3?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

I tested the extension and it seems to be pretty cool 🔥 

I have some feedback and suggestions for you:

- The command flickers when entering if you have anything on the list already, could you look into this? You can read a bit more about [empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states) here

- If wrong credentials are added, should we consider showing a toast saying that it's an invalid token?

- Maybe trim spaces in the token?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/shell-buddy/package.json (hunk 1 above)
Comment (JohnBra) [reply extensions/shell-buddy/package.json:17]: I interpreted the subtitle to name the "service it integrates with". The command conversion uses GPT-3 AI under the hood. I renamed it to "Shell Buddy" - I think that makes more sense anyway :)


Xnapper-2022-10-11-08 57 37

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**